docs/FHIR-anonymization.md

```diff
@@ -441,96 +473,128 @@ To generalize dateTime, time, date and instant type using expression
 Generalization uses FHIRPath predicate expression to define a set of cases that specify the condition and target value like [sample rules](#Sample-rules-using-FHIRPath). Follows are some examples of cases.
 
 |Data Type|Cases|Explanation|Input data-> Output data|
 |-----|-----|-----|-----|
-|numeric|_"$this>=0 and $this<20": "20"_|Data fall in the range [0,20) will be replaced with 20. |18 -> 20|
-|numeric|_"true": "($this div 10)*10"_|Approximate data to multiples of 10. |18 -> 10|
-|string| _"$this in ('es-AR' \| 'es-ES' \| 'es-UY')": "'es'"_|Data fall in the value set will be mapped to "es".|'es-UY' -> 'es'|
-|string| _"$this.startsWith(\'123\')": "$this.subString(0,2)+\'*\*\*\*\' "_ |Mask sensitive string code.|'1230005' -> '123****'|
-|date, dateTime, time|_"$this >= @2010-1-1": "@2010"_|Data fall in a date/time/dateTime range will be mapped to one date/time/dateTime value.| 2016-03-10 -> 2010|
-|date, dateTime, time|"$this.replaceMatches('(?&lt;year&gt;\\\d{2,4})-(?&lt;month&gt;\\\d{1,2})-(?&lt;day&gt;\\\d{1,2})\\\b', '${year}-${month}'"|Omit "day" to generalize specific date.|2016-01-01 -> 2016-01|
+|numeric|`"$this>=0 and $this<20": "20"`|Data fall in the range [0,20) will be replaced with 20. |18 -> 20|
+|numeric|`"true": "($this div 10)*10"`|Approximate data to multiples of 10. |18 -> 10|
+|string| `"$this in ('es-AR' \| 'es-ES' \| 'es-UY')": "'es'"`|Data fall in the value set will be mapped to "es".|'es-UY' -> 'es'|
+|string| `"$this.startsWith(\'123\')": "$this.subString(0,2)+\'*\*\*\*\' "` |Mask sensitive string code.|'1230005' -> '123****'|
+|date, dateTime, time|`"$this >= @2010-1-1": "@2010"`|Data fall in a date/time/dateTime range will be mapped to one date/time/dateTime value.| 2016-03-10 -> 2010|
+|date, dateTime, time|`"$this": "$this.toString().replaceMatches('(?<year>[0-9]{2,4})-(?<month>[0-9]{1,2})-(?<day>[0-9]{1,2})', '${year}-${month})'"`|Omit "day" to generalize specific date.|2016-01-01 -> 2016-01|
 
 For each generalization rule, there are several additional settings to specify in configuration files:
-- [required] **cases** An object defining key-value pairs to specify case condition and replacement value using FHIRPath predicate expression. _key_ represents case condition and _value_ represents target value.
 
+- [required] **cases** An object defining key-value pairs to specify case condition and replacement value using FHIRPath predicate expression. _key_ represents case condition and _value_ represents target value.
 - [optional] **otherValues** Define the operation for values that do not match any of the cases. The value could be "redact" or "keep". The default value is "redact".
 
 Since the output of FHIR expression is flexible, users should provide expressions with valid output value to avoid unexcepted errors.
 
+> **[Important]**:
```

Comment:
We can make these two points as examples for line 573  "users should provide expressions with valid output value to avoid unexcepted errors.", rather than an "Important" tag here.

Reply:
ok, I can adjust this.